### PR TITLE
New gHack: Intro to GKE

### PR DESCRIPTION
**Pull request by gfilicetti**
New hack is rendered at this url:

https://gfilicetti.github.io/cloud-hackathons/hacks/intro-to-gke/

### CODE REVIEW

File: hacks/intro-to-gke/README.md [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+   - Using your containerized app in Artifact Registry, deploy it to GKE and expose it to the public internet
+- Challenge 4: Scale the Application to Handle Increased Traffic
+   - Now that the application is deployed and out there, we've noticed an increase in traffic and need to scale out to handle the new load
+- Challenge 5: Update and Release with Zero Downtime
+   - Change is inevitable, but new releases need to be deployed smoothly. Here we learn how to do that with zero downtime
+
+## Prerequisites
+
+- A basic understanding of Docker and Kubernetes 
+   - If you lack a basic understanding, you can review [Docker](https://docs.docker.com/) and [Kubernetes](https://kubernetes.io/docs/home/) now.
```
Comment (meken) [inline hacks/intro-to-gke/README.md:44]: Shouldn't we add "access to an environment with gcloud & kubectl installed" and mention that CloudShell already has that?

File: hacks/intro-to-gke/README.md [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+
+### Introduction
+
+Now that our cluster is ready for action, we need to work on our monolithic application. Currently it is a stand alone node.js/React application that runs on the command line. We will be going through the steps to make this ready to deploy onto our Kubernetes cluster.
+
+### Description
+
+In this challenge, you will begin by cloning the application's git repository and making sure it works standalone first, before starting any work on it.
+
+You will need to clone its git repository with this command:
```
Comment (meken) [inline hacks/intro-to-gke/README.md:87]: Does it have to be cloned? Can't you just provide a link that'll get them the zip?

File: hacks/intro-to-gke/qwiklabs.yaml [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+schema_version: 2
+default_locale: en
+title: "[gHacks] Intro to GKE: Deploy, Scale and Update"
+description: ""
```
Comment (meken) [inline hacks/intro-to-gke/qwiklabs.yaml:17]: Copy the description from the main page here as well

File: hacks/intro-to-gke/README.md (hunk 1 above)
Comment (gfilicetti) [reply hacks/intro-to-gke/README.md:44]: Fixed, copied from your Cloud Run prereqs.

File: hacks/intro-to-gke/qwiklabs.yaml (hunk 3 above)
Comment (gfilicetti) [reply hacks/intro-to-gke/qwiklabs.yaml:17]: Good catch.

File: hacks/intro-to-gke/README.md (hunk 2 above)
Comment (gfilicetti) [reply hacks/intro-to-gke/README.md]: Ok, I've changed everything necessary in student and coach guide to refer to handling the zip instead of git repo

**Review by meken — APPROVED** [review]

**Pull request merged**